defaults/main.yml

```
@@ -425,3 +486,64 @@ galaxy_systemd_env: []
 
 # A list of additional python packages to install into galaxy's virtual environment
 galaxy_additional_venv_packages: []
+
+# galaxy themes variables
+galaxy_themes_subdomains: {}
+# - name: assembly
+#   tool_sections:
+#     - "hicexplorer"
+#     - "graph_display_data"
+#     - "peak_calling"
+#     - "assembly"
+#     - "annotation"
+#     - "genome_diversity"
+#     - "multiple_alignments"
+#   extra_tool_labels:
+#     - "proteomics"
+galaxy_themes_static_keys:
+  static_dir: ""
+  static_images_dir: "images/"
+  static_scripts_dir: "scripts/"
+  static_welcome_html: "welcome.html/"
```

```suggestion
  static_welcome_html: "welcome/"
```

maybe we can remove the extension to make it more clear it's a folder, I was reviewing this again and got confused again

---

but the folder is named automatically by Galaxy, I think removing this will break it.
https://github.com/galaxyproject/galaxy/blob/41b302de53e2d256dedefbdefd65f3709b6d85ee/lib/galaxy/webapps/base/webapp.py#L1186
https://github.com/galaxyproject/galaxy/blob/41b302de53e2d256dedefbdefd65f3709b6d85ee/lib/galaxy/web/framework/middleware/static.py#L34

---

Ohhh :skull:

Ok, guess we can't change that.